### DISCUSSION TITLE
How add flexfec to video stream in to example `rtp-to-webrtc`

### DISCUSSION THREAD

**Discussion by IgorAfanasiev** [pion/webrtc#2732] [Q&A]
I'm trying to start the example `rtp-to-webrtc` from this repository. And I would like to add Flexfec to a video stream. Is there any example? or can someone help me with how to do it? Thanks

**Comment by Sean-Der** [comment]
Hey @pougetat any advice/guidance on this? 

I am not the author of the FlexFEC code so don't know the answer off the top of my head :/
